tests/DCPS/C++11/Messenger/run_test.pl

```diff
@@ -20,4 +20,4 @@
 $test->process('Publisher', 'Publisher/publisher');
 $test->start_process('Publisher');
 
-exit $test->finish(30);
+exit $test->finish(120);
```

Comment:
Not sure change in timeout is needed, but maybe.
Although not directly related to this PR. maybe it's time to fix the GitHub Actions test failure due to
```
2021-05-20 16:22:29.540@LM_ERROR@(2756|5924) ERROR: Feature ownership_profile disabled, therefore history must be KEEP_LAST, depth 1.
```